From fd988d09003344009a51c73d11e629f19525319d Mon Sep 17 00:00:00 2001
From: Arnon Kaufman <akaufman@habana.ai>
Date: Mon, 19 Apr 2021 09:00:39 +0300
Subject: [PATCH] [SW-31823] i2cs: support specs uint32_t typedef

To support specs uintXX_t integers typedefs.
After adding the proto_status data structure to the hl_fw_if.h
file.

Change-Id: I949861d15d54ad5f061be533f8f145520da6bfe8
Signed-off-by: Arnon Kaufman <akaufman@habana.ai>
---

diff --git a/common/boot_desc.c b/common/boot_desc.c
index e8550d1..a688e82 100644
--- a/common/boot_desc.c
+++ b/common/boot_desc.c
@@ -4,6 +4,7 @@
  * All Rights Reserved.
  *
  */
+#include <stdint.h>
 #include <string.h>
 #include <stdbool.h>
 #include <preboot.h>
